### PR TITLE
Completely renovated webapp files. Upload images, pages, and updated …

### PR DESCRIPTION
**Pull request by erikard**
…css and html code for several new pages as well. Complete visual and flow change to previous version.

### CODE REVIEW

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -7,14 +7,25 @@
...
+          <li><a href='pages/Pictures.html'>Pictures</a></li>
+          <li><a href='pages/ContactMe.html'>Contact Me</a></li>
+          <li><a href='pages/PlayMe.html'>Interactive</a></li>
+      </ul>
+    </nav>
+    
+    <div style="margin-top:25px">
+      <h1>Welcome<br></h1>
+      <div style="margin-left:40px;margin-right:40px">
+      <p><br>So happy to see you here! Please feel free to explore and learn a little bit more about me and my work.<br><br></p>
```
Comment (rjkroege) [inline portfolio/src/main/webapp/index.html]: Can you line break this at something closer to 80 columns? I tend to be a fan of the [Google standard format](https://google.github.io/styleguide/htmlcssguide.html) where line-breaking is optional but I strongly support.

File: portfolio/src/main/webapp/index.html [hunk 2]
```diff
@@ -7,14 +7,25 @@
...
+        <img id="aboutMe" src="images/rix_trees.jpeg" alt="Girl with nature background">
+            <p> <br><br>I'm Rix, and I'm an undergraduate student pursuing a Computer Science major and a Mathematics minor at Texas Christian University. I'm a rising Junior, and I'm a STEM Scholar there-- and I'm super involved on campus! I'm a proud Charter sister of Kappa Delta Chi, and love to (1) Learn about technology, and (2) empower women. I love it most when both of these overlap!
```
Comment (rjkroege) [inline portfolio/src/main/webapp/index.html]: There's a HTML character entity (`&mdash;`) for a typographic "em" dash instead of typing --. It will stop the browser from (for example) deciding to line-break between the `-` characters.

Comment (rjkroege) [reply portfolio/src/main/webapp/index.html]: Also: the benefit of more line breaking is that it makes it easier to associate a comment with a specific line.

File: portfolio/src/main/webapp/index.html (hunk 1 above)
Comment (rjkroege) [reply portfolio/src/main/webapp/index.html]: In particular, the Google standard format recommends putting a tag per line.

Comment (rjkroege) [inline portfolio/src/main/webapp/index.html]: You are using `<br>` elements for spacing. It's much preferable to use CSS to control how you want things spaced. In this particular case, it looks like you might want to use the padding facility?

File: portfolio/src/main/webapp/pages/AboutMe.html [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+        <div>
+            <p style="color:#878787; text-indent: 30px; text-align:center"><br><i>Who am I? A multitasking lover, passionate for empowering women, and a fashionista. A baker, cook, and avid reader who enjoys nature walks and loves ice cream!</i> <br><br></p>
+            <p> I was born in Mexico and moved to the United States for Middle and High School. I'm currently a student at Texas Christian University, studying Computer Science with a Mathematics Minor. I'm currently also debating wether I should switch my major to Data Science... <br> <br><br></p>
+            </div>
+            <img id="aboutMe" src="../images/rix_kids.JPG" width="300" height= 100% >
+            <div class="desc">My sister and I, when we were younger</div>
+            
+            <h2 id="maintext" style="font-size: 17px"><br><br>I am extremely involved on campus at TCU. Some of my involvement includes: </h2>
+            <div>
+                <strong> Kappa Delta Chi Prospective Colony:</strong> President of a new sorority chapter at TCU <p id="maintext">A new multicultural chapter at TCU; looking to promote Unity, Honesty, Integrity, and Leadership and serve our communities. </p>
```
Comment (rjkroege) [inline portfolio/src/main/webapp/pages/AboutMe.html]: Avoid spaces between the end of a tag and the intended contents of the tag.

Comment (rjkroege) [reply portfolio/src/main/webapp/pages/AboutMe.html]: Also: perhaps consider a definition list here instead?

File: portfolio/src/main/webapp/style.css [hunk 4]
```diff
@@ -1,9 +1,91 @@
...
 }
 
-#greeting-container {
-  margin-top: 20px;
+body {
+    background-color: whitesmoke;
+    text-decoration-color: white;
+    font-family: arial;
+}
+nav {
```
Comment (rjkroege) [inline portfolio/src/main/webapp/style.css:20]: Use a consistent space convention: most stanzas are separated by blank lines.

**Review by rjkroege — CHANGES_REQUESTED** [review]

File: portfolio/src/main/webapp/index.html [hunk 5]
```diff
@@ -1,13 +1,12 @@
 <!DOCTYPE html>
 <html>
-  <head>
+
```
Comment (rjkroege) [inline portfolio/src/main/webapp/index.html]: Removing the `<head>` and `<body>` tags makes this a non-conforming HTML file. Browsers do their best with non-standard HTML but it's preferable to write conforming HTML because it parses faster and is more likely to look the same in different browsers.

Something like[HTML Tidy](https://www.html-tidy.org) can check your HTML for correctness.

**Review by rjkroege — CHANGES_REQUESTED** [review]
Nice increased usage of CSS but I don't understand the reason for some of your HTML edits.

File: portfolio/src/main/java/com/google/sps/servlets/GreetingServlet.java [hunk 6]
```diff
@@ -0,0 +1,29 @@
+package com.google.sps.servlets;
+
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/** Handles requests sent to the /hello URL. Try running a server and navigating to /hello! */
```
Comment (rjkroege) [inline portfolio/src/main/java/com/google/sps/servlets/GreetingServlet.java:9]: It would have been better to put the servlet file in a different PR.

File: portfolio/src/main/java/com/google/sps/servlets/GreetingServlet.java [hunk 7]
```diff
@@ -0,0 +1,29 @@
...
+@WebServlet("/greeting")
+public class GreetingServlet extends HttpServlet {
+    
+    /*
```
Comment (rjkroege) [inline portfolio/src/main/java/com/google/sps/servlets/GreetingServlet.java:13]: In general, you should avoid checking in commented out code. The point of version control (e.g. Git) is that you can just delete code that you (previously) checked in and get it back.

**Review by rjkroege — CHANGES_REQUESTED** [review]
You should not check in `.DS_Store` files. Please add a `.gitignore` to exclude them and remove here.  Your HTML looks very good now.

File: portfolio/src/main/java/com/google/sps/servlets/GreetingServlet.java (hunk 6 above)
Comment (erikard) [reply portfolio/src/main/java/com/google/sps/servlets/GreetingServlet.java:9]: Thank you! It didn't occur to me to do this at the time. This would absolutely help, because then my html can be incorporated before "moving on"... (I decided to work ahead but can definitely see why that would have been advantageous... I will do that in the future!)

File: portfolio/src/main/java/com/google/sps/servlets/GreetingServlet.java (hunk 7 above)
Comment (rjkroege) [reply portfolio/src/main/java/com/google/sps/servlets/GreetingServlet.java:13]: Let me be clearer. Remove the commented out code.

**Review by rjkroege — APPROVED** [review]
